src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for Roboto in case it fails to load. For example, `font-family: Roboto, Arial, sans-serif;` ensures that there will be a similar font displayed if Roboto is not available.